### PR TITLE
WIP: Initial lump commit for Russian translations

### PR DESCRIPTION
**Pull request by 4min**
Almost half of the UI is translated, and I now need some feedback on where we want to take it. I'm leaning towards a more relaxed and fun language, similar to UI of sites like habrahabr.ru and leprosorium.ru. Rancher's original UI looks fun, but texts are a little more enterprisey and unwieldy at times. Perhaps we could rewrite the English translations, too? 8)

P.S. There are many terms that we'll need to agree on translations, Docker rommunity in Russia is not that big, so some things we would need to make up. I think it would be cool to have a channel in Slack to discuss this instead of going back-and-forth via PRs. Can you do that?

### CODE REVIEW

**Comment by 4min** [comment]
BTW, please don't merge this one in, I haven't really finalized this, will keep working on this and rebase in my fork before sending in a more mature version for actual pull.

**Comment by vincent99** [comment]
We have a couple employees who speak Russian so @alena1108 can probably help you pick translated terms next week.  We occasionally have a bit of 'fun' here or there but in general it is not terribly appropriate for a tool large businesses are using (and sometimes paying for).

We don't currently have a (public) Slack, but are on IRC, #rancher in Freenode.

File: translations/ru-ru.yaml [hunk 1]
```diff
@@ -0,0 +1,1279 @@
...
+            =1 {одной организации}
+            other {# организациям}
+          } и {users, plural,
+            =0 {ни одному пользователю}
+            =1 {одному пользователю}
+            other {# пользователям}
+          }.
+        unrestricted: "Система {appName} настроена предоставлять доступ любому пользователю {github}а."
+      disabled:
+        label: 'Аунтификация черех ГитХаб не настроена'
```
Comment (alena1108) [inline translations/ru-ru.yaml]: Аутентификация

File: translations/ru-ru.yaml [hunk 2]
```diff
@@ -0,0 +1,1279 @@
...
+  edit: Редактировать
+  execute: Execute Shell
+  exportConfig: Экспортировать конфиг
+  finishUpgrade: Закончить обновление
+  logs: Смотреть логи
+  machineConfig: Конфиг машины
+  purge: Прочистить
+  remove: Удалить
+  restart: Перезапустить
+  restore: Восстонавить
```
Comment (alena1108) [inline translations/ru-ru.yaml]: Minor spelling mistake: восстАновить

File: translations/ru-ru.yaml [hunk 3]
```diff
@@ -0,0 +1,1279 @@
...
+  clone: Клонировать
+  console: Открыть консоль
+  deactivate: Деактивировать
+  edit: Редактировать
+  execute: Execute Shell
+  exportConfig: Экспортировать конфиг
+  finishUpgrade: Закончить обновление
+  logs: Смотреть логи
+  machineConfig: Конфиг машины
+  purge: Прочистить
```
Comment (alena1108) [inline translations/ru-ru.yaml]: This is the hard one...прочистить? очистить? вычистить? @imikushin?

File: translations/ru-ru.yaml [hunk 4]
```diff
@@ -0,0 +1,1279 @@
...
+    themeLight: День
+    themeAuto: Авто
+    themeDark: Ночь
+
+##############################
+# Resource Actions
+##############################
+action:
+  activate: Активировать
+  backup: Забекапить
```
Comment (alena1108) [inline translations/ru-ru.yaml]: if deleteBackup "Удалить копию", то сюда просится "сделать копию" или "скопировать".

File: translations/ru-ru.yaml [hunk 5]
```diff
@@ -0,0 +1,1279 @@
...
+  restoreFromBackup: Восстановить раздел из резервной копии
+  setDefault: Установить по умолчанию
+  snapshot: Снепшот
+  start: Запустить
+  startServices: Запустить сервисы
+  stop: Остановить
+  stopServices: Остановить сервисы
+  switchNamespace: Переключиться в это пространстро имён
+  switchEnvironment: Переключиться в это окружение
+  upgrade: Обновление
```
Comment (alena1108) [inline translations/ru-ru.yaml]: We should finalize whether we are gonna use verbs or nouns. Обновление vs Запустить

File: translations/ru-ru.yaml [hunk 6]
```diff
@@ -0,0 +1,1279 @@
...
+  viewConfig: Открыть конфиг
+  viewGraph: Открыть график
```
Comment (alena1108) [inline translations/ru-ru.yaml]: Граф. График это что-то другое

File: translations/ru-ru.yaml [hunk 7]
```diff
@@ -0,0 +1,1279 @@
...
+  cancelRollback: Отменить откат
+  clone: Клонировать
+  console: Открыть консоль
+  deactivate: Деактивировать
+  edit: Редактировать
+  execute: Execute Shell
+  exportConfig: Экспортировать конфиг
+  finishUpgrade: Закончить обновление
+  logs: Смотреть логи
+  machineConfig: Конфиг машины
```
Comment (alena1108) [inline translations/ru-ru.yaml]: as we refer to machine as "драйвера машин" on the lines 721 and 1227, do you think we should say "конфиг драйвера машин" здесь?

**Comment by vincent99** [comment]
@4min @alena1108 @imikushin We're looking at switching to Crowdin for translations if you want to try it out and let us know if it's easier to work with:  http://translate.rancher.com/project/rancher-ui/invite

**Comment by vincent99** [comment]
Merged via #837, please use http://translate.rancher.com for further updates

**Comment by 4min** [comment]
I was hoping you wouldn't merge this - it had way too many errors and no stylistic coherence. Is there a way to undo it so that I can provide an updated version this weekend? Or do we need to roll forward from here?

**Comment by vincent99** [comment]
I can remove it if you really want, though some translation seems better than none..

But the main point is to work through crowdin (translate.rancher.com)  instead of PRs (though you can still edit a yaml file locally and upload it to them at the end if you want)

**Pull request closed without merging**